examples/spec/lcdproc/LCDd.ini

```diff
@@ -0,0 +1,2752 @@
+[]
+mountpoint = LCDd.conf
+;infos/plugins = dump validation enum range ipaddr path network type
+infos/plugins = dump validation enum
+
+;Until the max plugin error is not fixed, some specifications will be left commented out
+
+;=============================================================================
+;=============================================================================
+
+;Affecting all subsettings, eg curses/Contrast, CFontz/Contrast, etc.
+;------------------------------------
+
+[/_/contrast]
+type = unsigned_short
+check/type = unsigned_short
+check/range = 0-1000
+description = Set the initial contrast
+
+[/_/brightness]
+type = unsigned_short
+check/type = unsigned_short
+check/range = 0-1000
+
+[/_/offbrightness]
+type = unsigned_short
+check/type = unsigned_short
+check/range = 0-1000
+
+[/_/reboot]
+type = boolean
+check/type = boolean
+default = no
+description = Reinitialize the LCD`s BIOS normally you shouldn`t need this
```

Comment:
Two sentences?

Reply:
Copied from the original config was like this:
```
# Reinitialize the LCD's BIOS [default: no; legal: yes, no]
# normally you shouldn't need this
```
I removed the `[..]` part because it is actually visible in other metadata anyway.
After reevaluation I think it is still good to have it in the description.